java/pom.xml

```diff
@@ -103,6 +103,11 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-maven-plugin</artifactId>
+          <version>1.17</version>
```

Comment:
but keep this 1.15. Let's just move it up for now, but not increase the version.

Reply:
Ooops! sorry, my bad. Corrected.